packages/opencensus-instrumentation-grpc/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "rootDir": ".",
     "outDir": "build",
     "pretty": true,
+    "lib": ["es2016", "dom"],
```

Comment:
Why is `dom` needed here?

Reply:
It's needed to provide `XMLHttpRequest` type definition for protobufjs: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/dbabb10c626deb803e817f214bb495b65c4a7d4f/types/protobufjs/index.d.ts#L275

Reply:
OK, makes sense.